test/irb/test_ruby_lex.rb

```diff
@@ -29,20 +30,59 @@ def teardown
       restore_encodings
     end
 
-    def assert_indenting(lines, correct_space_count, add_new_line)
+    def calculate_indenting(lines, add_new_line)
       lines = lines + [""] if add_new_line
       last_line_index = lines.length - 1
       byte_pointer = lines.last.length
 
       context = build_context
       context.auto_indent_mode = true
+
       ruby_lex = RubyLex.new(context)
-      io = MockIO_AutoIndent.new([lines, last_line_index, byte_pointer, add_new_line]) do |auto_indent|
+      mock_io = MockIO_AutoIndent.new(lines, last_line_index, byte_pointer, add_new_line) do |auto_indent|
```

Comment:
The block passed to `MockIO_AutoIndent.new` is not used because `&assertion` is deleted from parameter

Reply:
🤦‍♂️ Because I've moved the assertion out to `assert_row_indenting` and this is not needed.